.vscode/launch.json

```diff
@@ -146,7 +148,8 @@
         "ZETAPUSH_LOCAL_DEV": "true",
         "ZETAPUSH_LOG_LEVEL": "silly",
         "COMETD_LOG_LEVEL": "info",
-        "ZETAPUSH_COMMANDS_LOG_LEVEL": "-vvvv"
+        "ZETAPUSH_COMMANDS_LOG_LEVEL": "-vvvv",
+        "TS_NODE_SKIP_IGNORE": "true"
```

Comment:
Is this environment variable needed for launching integration ?
This environment variable seems to not be for integration itself but for projects that are run by integration. Should it be provided in tests instead (the aim is that a developer can run all tests easily, other environment variables are not mandatory to be able to run tests) ?